src/legacy/ui/public/vis/editors/default/components/default_editor_agg_select.tsx

```diff
@@ -89,17 +93,17 @@ function DefaultEditorAggSelect({
       i18n.translate('common.ui.vis.defaultEditor.aggSelect.noCompatibleAggsDescription', {
         defaultMessage: 'The index pattern {indexPatternTitle} does not contain any aggregations.',
```

Comment:
I know that you didn't introduce that in this PR, but since we're anyway touching that file atm, to:

`The index pattern {indexPatternTitle} does not have any aggregatable fields.`

(And then also just delete the existing translations for it, so it will be retranslated in the next batch.)

Reply:
Done.